examples/spark-sensors-proto/src/main/scala/sensors/proto/MovingAverageSparklet.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sensors.proto
+
+import cloudflow.spark.sql.SQLImplicits.StringToColumn
```

Comment:
Can you explain what this does?

Reply:
It Converts $"col name" into a `org.apache.spark.sql.Column`. See this line
        .groupBy(window($"ts", "1 minute", "30 seconds"), $"src", $"gauge")